### ISSUE TITLE
Turn “Your Songs” into a custom playlist

### ISSUE THREAD

**Issue by IronMaltese** [snuffyDev/Beatbump#30]
“Your Songs” takes up a lot of space in the library section, and I usually just press “see all” if I want to see my favorite songs, so if “Your Songs” was combined into a playlist called “Favorites” it would really un-clutter the library section

**Comment by snuffyDev** [comment]
I’ll see what I can do, that’s a good idea. Probably could change the layout to be tabular, one tab favorites, one tab playlists.

Just need to ensure I implement it properly to avoid potential data loss. Probably a good time to add exporting the Library to JSON since I’ve been wanting to add that.

Good suggestion!

**Comment by IronMaltese** [comment]
I don't believe that this feature needs multiple tabs, since ideally this feature would have one section for playlists, and favorites would be one of those playlists

**Comment by IronMaltese** [comment]
I believe that #25 should have a tabular design, with three tabs, one for playlists, artists and albums

**Comment by IronMaltese** [comment]
![image](https://user-images.githubusercontent.com/89056889/158724559-d8adaab4-b62c-4cd7-9e1b-7faf17eff2c6.png)
This is what I was thinking for the tab-based design, except playlists would be selected by default
